test/run.test.js

```diff
@@ -698,6 +698,35 @@ test('should count resets', t => {
   })
 })
 
+test('should get onResponse callback invoked even when there is no body', t => {
```

Comment:
please use async/await instead of `then`.

Reply:
Done